### ISSUE TITLE
User isolation via task.json

### ISSUE THREAD

**Issue by trhumphries** [microsoft/mindaro#38]
I've noticed that tasks.json is updated with an isolateAs property when you run Bridge to Kubernetes: Configure.
Should we be adding tasks.json to .gitignore and have each user run the configure step to make everyone uses a unique value for this setting?

**Comment by pragyamehta** [comment]
@trhumphries thanks for reaching out! Yes, the isolate feature is provided so that each user can generate a unique "isolateAs" header and are able to debug without affecting the stable version of the app or disrupting other users debugging the app running in the shared namespace. All users collaborating and sharing the namespace must select "Yes" to the isolate question when configuring Bridge to Kubernetes on their machines, else they may end up disrupting other users of the shared namespace.

tl;dr Each user should generate a different isolateAs header if multiple users are sharing the same namespace in your Kubernetes cluster.

**Comment by philon-msft** [comment]
Also yes, typically .gitignore files exclude the whole .vscode directory. I see that's not always the case with the .gitignore files in our samples - we'll get those updated.

**Comment by trhumphries** [comment]
Thanks!

**Comment by saidimu** [comment]
@pragyamehta @philon-msft it would be amazing to have this fully automated for multiple users of a repo. The current requirement to manually edit `tasks.json` is a bit unfortunate. So is the `tasks.json` limitation in resolving `isolateAs` variables: https://code.visualstudio.com/docs/editor/variables-reference#_why-arent-variables-in-tasksjson-being-resolved
